app/views/service_providers/_form.html.erb

```diff
@@ -59,7 +59,7 @@
                                 protocol,
                                 class: 'usa-radio__input' %>
           <%= form.label ('identity_protocol_' + protocol).to_sym,
-                         protocol.titleize,
+                         titleize(protocol),
```

Comment:
adding this to a specific helper method allowed me to remove the acronyms from `config/initializers/inflections.rb` which was a blocker for zietwork because of the `SamlIdp` gem